flaskServer/restEndpoint.py

```diff
@@ -0,0 +1,82 @@
+from flask import Flask, render_template, url_for, request, redirect
+from flask_restful import Resource, Api, reqparse
+from werkzeug import secure_filename
+from flask_cors import CORS, cross_origin
+import os
+from eventClass.eventHandler import EventHandler
+from eventClass.aeroCubeEvent import ImageEvent, ResultEvent, AeroCubeEvent
+from eventClass.aeroCubeSignal import *
+from eventClass.bundle import Bundle
+from tcpService.tcpClient import TcpClient
+app = Flask(__name__)
+api = Api(app)
+CORS(app)
+
+app.config['UPLOAD_FOLDER'] = 'flaskServer/static/img/'
```

Comment:
In order to make the repository cloneable on the Jetson without much issue, perhaps we should try to stick to relative file paths. If so, we should sweep through all of the files (including ImP files) to make the appropriate changes. Something cool could be setting a "root" or "home" directory at the top-level directory of the repo that all Python packages have access to.